homeassistant/components/fitbit/sensor.py

```diff
@@ -581,11 +581,14 @@ async def async_setup_platform(
             refresh_cb=lambda x: None,
         )
         try:
-            await hass.async_add_executor_job(authd_client.client.refresh_token)
+            updated_token = await hass.async_add_executor_job(
+                authd_client.client.refresh_token
+            )
         except OAuth2Error as err:
             _LOGGER.debug("Unable to import fitbit OAuth2 credentials: %s", err)
             translation_key = "deprecated_yaml_import_issue_cannot_connect"
         else:
+            _LOGGER.info("Updated refresh token: %s", updated_token)
```

Comment:
```suggestion
```

Reply:
Thanks, that was meant to be debug, but probably not needed given fitbit library logging can capture this.